### ISSUE TITLE
Building on windows might result in qqq.json and en.json issues

### ISSUE THREAD

**Issue by carlosperate** [google/blockly#447]
This has been a general issue that seems to be somewhat intermittent, but been around quite a long time.

I've reported it on the forum a while ago (though at the time it was fixed, but I still see it from time to time): https://groups.google.com/forum/#!searchin/blockly/build$20windows/blockly/Rnp4GW8lGcU/2c_6_al8rIAJ

And you can see it sometimes happening to other uses PR submissions: https://github.com/google/blockly/pull/241 https://github.com/google/blockly/pull/248

Basically, for me personally only under windows, sometimes you get errors while parsing the `qqq.json` and/or `en.json` files. Causing the content of this files to not be generated. Reverting the file back in git and building again somehow seems to get it right on the second pass.

I didn't experience this in quite a long time, but I've seen it more often lately, I will keep an eye and capture the exact console output the next time it happens.

**Comment by MarkusBordihn** [comment]
Example error message:
```
C:\Users\***\Documents\GitHub\blockly [master +0 ~6 -0]> python .\build.py
Error reading .\msg\json\en.json
Traceback (most recent call last):
  File "i18n\create_messages.py", line 149, in <module>
    main()
  File "i18n\create_messages.py", line 65, in main
    source_defs = read_json_file(os.path.join(os.curdir, args.source_lang_file))
  File "C:\Users\***\Documents\GitHub\blockly\i18n\common.py", line 64, in read_json_file
    raise InputError(filename, str(e))
common.InputError: .\msg\json\en.json: Expecting property name enclosed in double quotes: line 9 column 1 (char 176)
('Error running i18n/create_messages.py: ', CalledProcessError())
```

**Comment by carlosperate** [comment]
Sounds exactly right, thanks @MarkusBordihn !

**Comment by jano42** [comment]
I solve it by _touching_ "msg/json/en.json"

**Comment by jasonc1025** [comment]
Thanks @carlosperate for documenting the same issue that I'm experiencing.

**Comment by carlosperate** [comment]
Closing as #1050 should fix this issue.
